eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -194,133 +142,101 @@ steps:
       . $(Build.Repository.LocalPath)\\scripts\\utils\\PostGitCommitStatus.ps1
       SetCommitStatusForTestResult -PersonalAccessToken $(NuGetLurkerPersonalAccessToken) -VstsPersonalAccessToken $(System.AccessToken) -CommitSha $(Build.SourceVersion) -TestName "$env:AGENT_JOBNAME"
   condition: "not(eq(variables['ManualGitHubChecks'], 'false'))"
-
-- task: PublishBuildArtifacts@1
```

Comment:
I see we define all the outputs in the pipeline.yml

Reply:
Yes, the outputs are part of the template now. Maybe it's possible to reduce duplication. I'll think about it after this PR.